modules/bitgo/test/v2/unit/tss/common.ts

```diff
@@ -152,3 +158,106 @@ describe('commonVerifyWalletSignature', function () {
     }
   });
 });
+
+describe('getChallengesForEcdsaSigning', function() {
+  const bitgo = TestBitGo.decorate(BitGo, { env: 'mock' });
+  const rawEntChallenge = {
+    nTilde: 'ent ntilde',
+    h1: 'ent h1',
+    h2: 'ent h2',
+  };
+  const rawBitGoChallenge = {
+    nTilde: 'bitgo ntilde',
+    h1: 'bitgo h1',
+    h2: 'bitgo h2',
+  };
+  const walletId = 'walletId';
+  const entId = 'entId';
+  let adminEcdhKey;
+  let fakeAdminEcdhKey;
+
+  before(function() {
+
+    adminEcdhKey = bitgo.keychains().create();
+    fakeAdminEcdhKey = bitgo.keychains().create();
+
+    sinon.stub(bitgo, 'getSigningKeyForUser').resolves({
+      userId: 'id',
+      userEmail: 'user@bitgo.com',
+      pubkey: adminEcdhKey.xpub,
+      path: 'm/0/0',
+      ecdhKeychain: 'my keychain',
```

Comment:
These two properties only exist if the user fetches their own key. 

If another user is fetching a users key it will only return the first three properties.